readme.md

```diff
@@ -219,7 +229,9 @@ Of the three options below, which is the most "correct" way of organizing the fi
   songs/
     form.html
 ```
-
+```
+Option A is the correct Angular format because it separates the Controller, Index and Form as the primary forks - and does not arrange the file structure in the style of a psql db system.
+```
```

Comment:
B is correct. Angular is made up of templates and components that are very closely-linked. In this layout, related components are listed right next to each other in alphabetical order.